mkosi.conf.d/20-opensuse.conf

```diff
@@ -0,0 +1,58 @@
+[Match]
+Distribution=opensuse
+
+[Content]
+Packages=
+        attr
+        bash-completion
+        bc
+        ca-certificates
+        coreutils
+        diffutils
+        findutils
+        fio
+        gawk
+        gdb
+        git-core
+        grep
+        gzip
+        hostname
+        hwloc
+        indent
+        kbd
+        kexec-tools
+        less
+        libcap-ng-utils
+        libelf1
+        lvm2
+        make
+        man
+        mtools
+        nano
+        nftables
+        nvme-cli
+        openssh-server
+        openssl
+        pam
+        psmisc
+        python3
+        rpm
+        sed
+        socat
+        strace
+        stress-ng
+        sudo
+        systemd
+        systemd-boot
+        systemd-network
+        tmux
+        tree
+        udev
+        util-linux
+        valgrind
+        zip
+        zstd
+
+BuildPackages=
+        kernel-devel
+        libcap-devel
```

Comment:
Is `kernel-devel` really needed? It just seems to consist out of the sources for the suse kernel. Also `libcap-devel` can be moved to `Packages=`

Reply:
Good catch, I think `kernel-devel` came from Christian's fork.